decidim/decidim
# Remove proposal show_my_location_button initializer

#### :tophat: What? Why?
Back in #13423 we have introduced an initializer that was meant to allow displaying the `use my location button` in other components than proposals. 
During the decidim maintainers meeting, the maintainers and product team agreed that the functionality doesn't make sense in other components. This PR removes the initializer. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13423

#### Testing
1. Create a development / test app 
2. See the initializer or the env variable is not being present 

:hearts: Thank you!
